amalgamate/crow_all.h

```diff
@@ -1214,7 +1214,7 @@ namespace crow
             Object,
         };
 
-        const char* get_type_str(type t) {
+        inline const char* get_type_str(type t) {
```

Comment:
Looks like sublime clipped a bunch of extra spaces. haha. This is the only important change. It's also reflected in the algamation.